packages/odyssey/src/scss/components/_button.scss

```diff
@@ -124,24 +124,27 @@
   }
 }
 
-.is-ods-button-overlay {
-  background-color: $white;
-  color: $text-body;
+.is-ods-button-dismiss {
+  padding: $spacing-xs-em / 2;
+  border: 0;
+  background-color: transparent;
+  color: inherit;
+  line-height: 1;
 
   &:hover,
   &.is-ods-button-hover {
     border-color: transparent;
-    background-color: cv('gray', '100');
+    background-color: transparentize($white, 0.6);
```

Comment:
Do we have a ticket to remove this? `transparentize` is going to be an issue when it comes to customization.

Reply:
I don't think we do. I haven't thought much about transparency + customization - would love to get your thoughts.